### PR TITLE
Lps 88439 2

### PR DESCRIPTION
**Pull request by arthurchan35**

### CODE REVIEW

File: modules/dxp/apps/saml/saml-impl/src/main/java/com/liferay/saml/util/internal/SamlSendAuthnRequestUtilImpl.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
+/**
```
Comment (liferay-appsec) [inline modules/dxp/apps/saml/saml-impl/src/main/java/com/liferay/saml/util/internal/SamlSendAuthnRequestUtilImpl.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/dxp/apps/saml/saml-web/src/main/java/com/liferay/saml/web/internal/portlet/action/SendAuthnRequestMVCResourceCommand.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
+/**
```
Comment (liferay-appsec) [inline modules/dxp/apps/saml/saml-web/src/main/java/com/liferay/saml/web/internal/portlet/action/SendAuthnRequestMVCResourceCommand.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/dxp/apps/saml/saml-web/src/main/java/com/liferay/saml/web/internal/portlet/action/SendSamlSloRequestInfosMVCResourceCommand.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
+/**
```
Comment (liferay-appsec) [inline modules/dxp/apps/saml/saml-web/src/main/java/com/liferay/saml/web/internal/portlet/action/SendSamlSloRequestInfosMVCResourceCommand.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/dxp/apps/saml/saml-hook/docroot/META-INF/custom_jsps/html/portal/saml/select_idp.jsp [hunk 4]
```diff
@@ -24,9 +24,11 @@ JSONObject samlSsoLoginContext = (JSONObject)request.getAttribute("SAML_SSO_LOGI
 JSONArray relevantIdpConnectionsJSONArray = samlSsoLoginContext.getJSONArray("relevantIdpConnections");
 %>
 
-<aui:form action='<%= PortalUtil.getPortalURL(request) + "/c/portal/login" %>' method="get" name="fm" style="padding: 1rem;">
-	<aui:input name="saveLastPath" type="hidden" value="<%= false %>" />
+<liferay-portlet:resourceURL id="/saml/send_authn_request" var="sendAuthnRequestURL" />
+
+<aui:form action="<%= sendAuthnRequestURL %>" id="fm" name="fm" style="padding: 1rem;">
 	<aui:input name="redirect" type="hidden" value="<%= redirect %>" />
```
Comment (liferay-appsec) [inline modules/dxp/apps/saml/saml-hook/docroot/META-INF/custom_jsps/html/portal/saml/select_idp.jsp:30]: This line adds a form input. Please check:

* The input is validated on the processing action

* The backend is protected agains value forging



Refer to documentation:

File: modules/dxp/apps/saml/saml-web/src/main/resources/content/Language.properties [hunk 5]
```diff
@@ -1,6 +1,7 @@
 a-further-x-entries-of-other-types-were-ignored=A further {0} entries of other types were ignored.
```
Comment (liferay-appsec) [inline modules/dxp/apps/saml/saml-web/src/main/resources/content/Language.properties:1]: Check that added properties are either titles or sentences

**Review by liferay-appsec — COMMENTED** [review]
This is an automatic review suggestion

**Comment by arthurchan35** [comment]
ci:test:relevant

**Comment by arthurchan35** [comment]
ci:test:sf

**Pull request closed without merging**